assets/pages/regular-contributions-existing/regularContributionsExisting.jsx

```diff
@@ -22,31 +24,33 @@ pageInit();
 
 // ----- Render ----- //
 
+
 const content = (
   <div className="gu-content">
     <SimpleHeader />
+    <CirclesIntroduction
+      headings={['Whoops!']}
+    />
+    <div className="multiline-divider" />
+
     <section className="existing gu-content-filler">
       <div className="existing__content gu-content-filler__inner">
-        <div className="existing__wrapper">
-          <h1 className="existing__heading">Whoops!</h1>
-          <h2 className="existing__subheading">
-            Looks like you are already making a regular contribution to the
-            Guardian - thank you. If you&#39;re feeling generous, there is
-            another way you can&nbsp;help.
-          </h2>
-          <CtaLink
-            ctaId="contribute-one-off-again"
-            text="Make a one-off contribution"
-            url={routes.oneOffContribCheckout}
-            accessibilityHint="Further support the guardian over and above your current regular contribution"
-          />
+        <div className="component-existing-contribution">
```

Comment:
I'm not sure we've done a particularly good job of documenting this anywhere, but under our conventions this would probably have the class `existing-contribution`.

Components are usually explicitly independent [React components](https://reactjs.org/docs/react-component.html), which in this codebase we implement as [stateless functional components](https://reactjs.org/docs/components-and-props.html#functional-and-class-components). We combine this with BEM by treating the top-level element in the React component as the "block" in BEM, and applying classes to everything within accordingly. That way, all styles associated with that component can be scoped to it via these classes.

In this case the `<div>` is just another element sitting within the page, and not an independent component in itself.

Alternatively you can drop this `<div>` entirely and rely on the fact that the `<PageSection>` below will have the unique modifier class `component-page-section--existing-contribution`, although this raises the question of what class you use on the `<p>` tag below 🤔. Possibly that could have a one-off class of something like `existing-contributor-copy`... it's not perfect but sometimes things don't quite fit into our componentised-BEM structure 😔.